api/src/test/java/com/basistech/rosette/api/RosetteAPITest.java

```diff
@@ -96,16 +99,10 @@ public void setUp() throws IOException, InterruptedException, RosetteAPIExceptio
             language = LanguageCode.UNKNOWN;
         }
 
-        String statusFilename = testFilename.replace(".json", ".status");
-        try (InputStream bodyStream = new FileInputStream("src/test/mock-data/response/" + testFilename)) {
+        try (InputStream bodyStream = new FileInputStream("src/test/mock-data/response/doc-data.json")) {
```

Comment:
Can you retire this as well? The json in the file is simple enough to be a Java string here.